modules/lazyload/assets/js/frontend.js

```diff
@@ -0,0 +1,21 @@
+window.addEventListener( 'DOMContentLoaded', () => {
+    const dataAttribute = 'data-e-bg-lazyload-src';
+    const lazyloadBackgrounds = document.querySelectorAll( `[${ dataAttribute }]:not(.lazyloaded)` );
```

Comment:
How can it be `lazyloaded` on `DOMContentLoaded`

Reply:
done